### PR TITLE
fix(benchmark): Fix args formatting & validate scaling mode env vars

### PR DESCRIPTION
**Pull request by tomi**
## Summary

- Fix formatting of cli args 
- Validate scaling mode env vars 

## Review / Merge checklist

- [ ] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [ ] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs [hunk 1]
```diff
@@ -36,6 +36,14 @@ async function main() {
 	const vus = argv.vus;
 	const duration = argv.duration;
 
+	const hasN8nLicense = !!n8nLicenseCert || !!n8nLicenseActivationKey;
+	if (n8nSetupToUse === 'scaling' && !hasN8nLicense) {
+		console.error(
+			'N8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```
Comment (csuermann) [inline packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]:
```suggestion
			'n8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```

Comment (csuermann) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: we always refer to our company as "n8n" (all lower-case)

**Review by csuermann — DISMISSED** [review]

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs (hunk 1 above)
Comment (tomi) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: Tru dat, copilot should learn that! 😄

**Review by csuermann — APPROVED** [review]

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs [hunk 2]
```diff
@@ -36,6 +36,14 @@ async function main() {
 	const vus = argv.vus;
 	const duration = argv.duration;
 
+	const hasN8nLicense = !!n8nLicenseCert || !!n8nLicenseActivationKey;
```
Comment (ivov) [inline packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:39]: Shoudl we check for tenant ID as well?

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs [hunk 3]
```diff
@@ -36,6 +36,14 @@ async function main() {
...
+	if (n8nSetupToUse === 'scaling' && !hasN8nLicense) {
+		console.error(
+			'n8n license is required to run the scaling setup. Please provide N8N_LICENSE_CERT or N8N_LICENSE_ACTIVATION_KEY',
```
Comment (ivov) [inline packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: Strictly speaking scaling mode doesn't require a license, only multi-main does.

Comment (tomi) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: Isn't scaling mode == multi-main?

Comment (ivov) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: Scaling mode is the Bull-based setup, which comes in single-main (free) and multi-main (licensed) flavors.

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs (hunk 2 above)
Comment (tomi) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:39]: I think that's not necessarily required, but only for certain licenses, so I left it out

File: packages/@n8n/benchmark/scripts/runForN8nSetup.mjs (hunk 3 above)
Comment (tomi) [reply packages/@n8n/benchmark/scripts/runForN8nSetup.mjs:42]: Right, and queue mode is the legacy name for scaling mode? I think we should clarify the naming.

**Comment by Joffcom** [comment]
Got released with [`n8n@1.59.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.59.0)

**Pull request merged**